pallets/omnipool/src/tests/mock.rs

```diff
@@ -137,6 +155,20 @@ parameter_type_with_key! {
 	};
 }
 
+//NOTE: oracle is not used in the unit tests. It's here to satify benchmarks bounds.
+use pallet_ema_oracle::MAX_PERIODS;
+parameter_types! {
+	pub SupportedPeriods: BoundedVec<OraclePeriod, ConstU32<MAX_PERIODS>> = BoundedVec::truncate_from(vec![
+		OraclePeriod::LastBlock, OraclePeriod::Short, OraclePeriod::TenMinutes]);
+}
+impl pallet_ema_oracle::Config for Test {
+	type Event = Event;
+	type WeightInfo = ();
+	type BlockNumberProvider = MockBlockNumberProvider;
+	type SupportedPeriods = SupportedPeriods;
+	type MaxUniqueEntries = ConstU32<20>;
+}
```

Comment:
i dont see why this is needed at all. since it is not used for the test.

Reply:
because of benchmarks tests. Benchmarks needs oracle in runtime.